packages/block-editor/package.json

```diff
@@ -41,6 +41,7 @@
 		"@wordpress/keycodes": "file:../keycodes",
 		"@wordpress/priority-queue": "file:../priority-queue",
 		"@wordpress/rich-text": "file:../rich-text",
+		"@wordpress/shortcode": "file:../shortcode",
```

Comment:
ESLint error blocking merge:

`42:1  error  '@wordpress/shortcode' should be listed in the project's dependencies. Run 'npm i -S @wordpress/shortcode' to add it`